app/src/initializeApp.js

```diff
@@ -37,7 +37,16 @@ function setOptions(urlParams, store) {
   }
 }
 
+export const bootError = (error) => {
+  document.querySelector("#app").innerText = `
+    Lunie failed to start. Please contact us at http://help.lunie.io/
```

Comment:
i'm not sure if the help center is the right place to send people.

Reply:
where should we send them? email?

Reply:
would be nice to have a link that opens intercom automagically - but i guess until then we don't have to send them anywere. yeah we can add a support email.

Reply:
Help Center has Intercom

Reply:
it's not really obvious / in the corner - but it's ok - this is a rare screen 😄